setup.cfg

```diff
@@ -20,10 +20,12 @@ license_files = LICENSE
 # TODO(xuanwn): include all files in src/python/grpcio/grpc
 [pytype]
 inputs =
-    src/python/grpcio/grpc/experimental
```

Comment:
This seems to be removing PyType checking from several modules that already have it enabled? Why?

Reply:
Added a TODO for myself to account for all packages before merging feature branch back to master. also created a bug for same and assigned to myself

Reply:
Done .